src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPClientFeatures.java

```diff
@@ -145,6 +147,18 @@ public boolean isCaseSensitive(@NotNull PsiFile file) {
         return false;
     }
 
+    /**
+     * Returns the language grammar-specific statement terminator characters for the file.
+     *
+     * @param file the file
+     * @return the language grammar-specific statement terminator characters for the file
+     */
+    @NotNull
+    public String getStatementTerminatorCharacters(@NotNull PsiFile file) {

```

Comment:
Default accessor for language grammar-specific statement terminator characters. Empty if unspecified.